Stripe/STPShippingAddressViewController.m

```diff
@@ -75,6 +75,7 @@ - (instancetype)initWithConfiguration:(STPPaymentConfiguration *)configuration
         }
         else if (prefilledInformation != nil) {
             STPAddress *prefilledAddress = [STPAddress new];
+            prefilledAddress.country = _addressViewModel.address.country;
```

Comment:
Is this bug limited to the shipping screen or should we make the same fix for billing also?